rag/llm/cv_model.py

```
@@ -164,9 +162,7 @@ def _extract_text_from_content(content):
             for blk in content:
                 if not isinstance(blk, dict):
                     continue
-                if blk.get("type") in {"text", "input_text"} and blk.get("text"):
-                    texts.append(str(blk["text"]))
-                elif "text" in blk and isinstance(blk.get("text"), (str, int, float)):
+                if blk.get("type") in {"text", "input_text"} and blk.get("text") or "text" in blk and isinstance(blk.get("text"), (str, int, float)):
```

Ruff (0.16.1)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)